### PR TITLE
Add changelog for v1.9.6

### PR DESCRIPTION
**Pull request by MarcosDY**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.9.6] - 2024-05-14
+
+### Added
+
+- Support for CGroups v2 in K8s and Docker workload attestors (#5076)
+- `gcp_cloudstorage` BundlePublisher plugin (#4961)
+- Allow to verify if the account ID is part of an AWS Organization in `aws_iid` node attestor (#4838)
+- Add more filtering options to entries and agents endpoints (#4714)
```
Comment (amartinezfayo) [inline CHANGELOG.md:10]:
```suggestion
- More filtering options to count and show entries and agents (#4714)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+
+### Changed
+
+- Credential composer to parse timestamps claims to integer (#5115)
+- FetchJWTBundles no longer return nil keys in favor of empty (#5031)
```
Comment (amartinezfayo) [inline CHANGELOG.md:15]:
```suggestion
- FetchJWTBundles now returns an empty collection of keys instead of null (#5031)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+
+### Fixed
+
+- Prevent to use expired tokens when connecting to database (#5119)
+- Server no longer try to create JWT authority when X.509 authority fails (#5064)
```
Comment (amartinezfayo) [inline CHANGELOG.md:20]:
```suggestion
- Server no longer tries to create JWT authority when X.509 authority fails (#5064)
```

**Review by amartinezfayo — COMMENTED** [review]
Thank you @MarcosDY!
I have some suggestions.

**Review by amartinezfayo — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.9.6] - 2024-05-14
+
+### Added
+
+- Support for CGroups v2 in K8s and Docker workload attestors (#5076)
```
Comment (azdagron) [inline CHANGELOG.md:7]:
```suggestion
- Opt-in support for CGroups v2 in K8s and Docker workload attestors (#5076)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,25 @@
...
+- `gcp_cloudstorage` BundlePublisher plugin (#4961)
+- The `aws_iid` node attestor can now check if the AWS account ID is part of an AWS Organization (#4838)
+- More filtering options to count and show entries and agents (#4714)
+
+### Changed
+
+- Credential composer to parse timestamps claims to integer (#5115)
```
Comment (azdagron) [inline CHANGELOG.md:14]:
```suggestion
- Credential composer to not convert timestamp related claims (i.e., exp and iat) to floating point values (#5115)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,25 @@
...
+- FetchJWTBundles now returns an empty collection of keys instead of null (#5031)
+
+### Fixed
+
+- Prevent to use expired tokens when connecting to database (#5119)
```
Comment (azdagron) [inline CHANGELOG.md:19]:
```suggestion
- Using expired tokens when connecting to database (#5119)
```

**Review by azdagron — APPROVED** [review]

**Review by amartinezfayo — APPROVED** [review]

**Pull request merged**